examples/graphics/conway_pretty.cpp

```diff
@@ -50,7 +52,7 @@ int main(int argc, char *argv[])
 
         array display = tile(state, 1, 1, 3, 1);
 
-        while(!simpleWindow.close() || !prettyWindow.close()) {
+        while(!simpleWindow.close() && !prettyWindow.close()) {
```

Comment:
Set this back to ||. The checks are being done before the image calls in line 58 and 59.